### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Url Rewrite- Outbound Rules_Keyboard: At Substitution value label focus is first going to macros button and then coming to the editing field 

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#151]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NET Websites; #Edge; #Chrome; #Keyboard; #MAS2.4.3

**Environment Details:**
URL: https://manage.iis.net/webserver/url-rewrite
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
Keyboard
MAS ID Violated: MAS 2.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Url Rewrite and press enter
6. Tab till outbound rules tab and press enter
7. Tab till Create Rule button and press enter
8. Tab till substitution value label. Verify at Substitution value label focus is first going to macros button and then coming to the editing field or not

**Actual Result:**
At Substitution value label focus is first going to macros button and then coming to the editing field
 
**Expected Result:**
At Substitution value label focus should first come to the editing field and then should go to the macros button

**User Impact:**
Disable users may find difficult while working with the application if focus first goes to the macros

**Attachments:**
[Bug151_outbound rules_focus order.pptx](https://github.com/Microsoft/IIS.WebManager/files/2330982/Bug151_outbound.rules_focus.order.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
